src/features/MoviesBrowser/MovieDetailPage/MovieHeader/styled.js

```diff
@@ -1,130 +1,53 @@
 import styled from "styled-components";
 
 export const Wrapper = styled.div`
-    max-width: 1920px;
-    height: 770px;
-
-    @media ( max-width: ${({ theme }) => theme.breakpoints.mobile}px) {
-    };
+    width: 100%;
+    background-color: ${({ theme }) => theme.colors.navBackground};
 `;
 
 export const WrapperPoster = styled.div`
-    height: 770px;
+    width: 100%;
+    max-width: ${({ theme }) => theme.breakpoints.container};
     margin: 0 auto;
     background-repeat: no-repeat;
     background-position: center;
-    background-size: contain;
-    
-    @media ( max-width: ${({ theme }) => theme.breakpoints.mobile}px) {
-    };
+    background-size: cover;
+    background-image: url('${({ image }) => image}');
+    box-shadow: inset 0 0 5vw 5vw ${({ theme }) => theme.colors.navBackground};
+    background-position: top;
 `;
 
 export const WrapperContent = styled.div`
-    width: auto;
-    height: 770px;
+    padding: 56px 0;
+    width: auto; 
+    height: 80vh;
     margin: 0 auto;
     display: flex;
     align-items: flex-start;
     justify-content: flex-end;
+    text-shadow:    0px 0px 15px ${({ theme }) => theme.colors.navBackground},
+                    0px 0px 7px ${({ theme }) => theme.colors.navBackground};
     flex-direction: column;
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: contain;
-    background: linear-gradient(270deg, #000000, 
-                rgba(0, 0, 0, 0.873268), 
-                rgba(0, 0, 0, 0.720529), 
-                rgba(0, 0, 0, 0.294577), 
-                rgba(0, 0, 0, 0.159921), 
-                rgba(0, 0, 0, 0) 25.68%), 
-                linear-gradient(180deg, #000000, 
-                rgba(0, 0, 0, 0.984059),
-                rgba(0, 0, 0, 0.967732),
-                rgba(0, 0, 0, 0.865569),
-                rgba(0, 0, 0, 0.230315), 
-                rgba(0, 0, 0, 0) 26.64%), 
-                linear-gradient(90deg, #000000, 
-                rgba(0, 0, 0, 0.689555),
-                rgba(0, 0, 0, 0.439033),
-                rgba(0, 0, 0, 0.20628),
-                rgba(0, 0, 0, 0) 24.22%), 
-                linear-gradient(180deg, 
-                rgba(0, 0, 0, 0) 58.48%, 
-                rgba(0, 0, 0, 0.106473), 
-                rgba(0, 0, 0, 0.235359), 
-                rgba(0, 0, 0, 0.492821), 
-                rgba(0, 0, 0, 0.740286), #000000 92%);
-
-    @media ( max-width: ${({ theme }) => theme.breakpoints.mobile}px) {
-    };
-`;
-
-export const Text = styled.article`
-    @media ( max-width: ${({ theme }) => theme.breakpoints.mobile}px) {
+    
+    @media (max-width: ${({ theme }) => theme.breakpoints.tabletLandscape}) {
+        height: auto;
+        width: 100%;
```

Comment:
I can see a lot of changes, but it's nice that the shadows display correctly :)

Reply:
The main problem was that the whole part should be out of `<Main>` component to reach all the width